### ISSUE TITLE
您好，我也是做这方面工作的。能否交流下？

### ISSUE THREAD

**Issue by CXMANDTXW** [microsoft/MaskFlownet#13]
您好，我也是做这方面工作的。Occlusion+optical flow，去年我就完成了预测occlusion mask放入PWC-Net网络作为decoder分支网络的辅助信息类似的工作，很遗憾后面没有就这个工作深入下去。
还没深入看您的paper，但感觉思路和我当时的相差无几，如果方便的话，能否留个联系方式探讨下？

**Comment by zsyzzsoft** [comment]
Would love to discuss if you have any questions. Feel free to leave comments via issues or emails.

**Comment by CXMANDTXW** [comment]
I am confused about the mask training process.In the paper:We answer this question positively by showing that the network can indeed learn to mask such areas without any explicit supervision.
I personally understand that training occlusion mask does not require the supervision information of the mask,it makes me feel strange.Why a convolution layer can train the&nbsp;occlusion mask without using the ground truth of occlusion?I&nbsp;see in the code,the network has used the ground truth of occlusion(e.g.,Sintel,KITTI),and&nbsp;for the chairs dataset, initialize the mask to 1 as&nbsp;the ground truth.However, the ground truth of occlusion is not used to calculate the loss function of occlusion mask, but to obtain the loss function of non occluded area of optical flow.
By the way,in my work, I use the ground truth of occlusion and the predicted mask of my network to calculate the cross entropy as the loss function.It doesn't look very good.
Please forgive me for my&nbsp;poor english.


Best wishes!




------------------&nbsp;原始邮件&nbsp;------------------
发件人:&nbsp;"zsyzzsoft"<notifications@github.com&gt;;
发送时间:&nbsp;2020年5月22日(星期五) 晚上10:11
收件人:&nbsp;"microsoft/MaskFlownet"<MaskFlownet@noreply.github.com&gt;;
抄送:&nbsp;"_____丶"<616559347@qq.com&gt;;"Author"<author@noreply.github.com&gt;;
主题:&nbsp;Re: [microsoft/MaskFlownet] 您好，我也是做这方面工作的。能否交流下？ (#13)





 
Would love to discuss if you have any questions. Feel free to leave comments via issues or emails.

**Comment by zsyzzsoft** [comment]
Yes, our main observation is that the occlusion mask can be learned without any explicit supervision. More details can be found in Section 3 of our paper.
